mne/viz/_3d.py

```diff
@@ -1227,19 +1230,29 @@ def _plot_sensors(renderer, info, to_cf_t, picks, meg, eeg, fnirs,
         if len(locs[sensor_type]) > 0:
             sens_loc = np.array(locs[sensor_type])
             sens_loc = sens_loc[~np.isnan(sens_loc).any(axis=1)]
-            color = defaults[sensor_type + '_color']
             scale = defaults[sensor_type + '_scale']
-            actor, _ = _plot_glyphs(renderer=renderer, loc=sens_loc * scalar,
-                                    color=color, scale=scale * scalar,
-                                    opacity=sensor_opacity,
-                                    orient_glyphs=orient_glyphs,
-                                    scale_by_distance=scale_by_distance,
-                                    project_points=project_points,
-                                    surf=surf, check_inside=check_inside,
-                                    nearest=nearest)
+            actor_list = []
+            for idx_sen in range(len(info["ch_names"])):
+                if sensor_colors is None:
+                    color = defaults[sensor_type + '_color']
+                else:
+                    if type(sensor_colors) == list:
```

Comment:
this is the preferred way to do type checking:

```suggestion
                    if isinstance(sensor_colors, list):
```

Reply:
...although requiring it to be a list is not ideal. Best would be if a list, tuple, or 1-D array all worked equally well.  A good way to do this is to take whatever the user input was (unless it's `None`) and cast it to a numpy array, then triage based on `array.ndim` or `array.shape`.